tests/acceptance/steps/watermark_get.py

```diff
@@ -31,3 +31,29 @@ def step_impl5(context):
     lowmark_pattern = 'Low Watermark: 0'
     assert highmark_pattern in context.output
     assert lowmark_pattern in context.output
+
+
+@when(u'we call the watermark_get command with -r')
+def step_impl6(context):
+    context.output = call_watermark_get('ab', regex=True)
```

Comment:
Can we change this to 'bc' instead?

Reply:
I would recommend to extend the topic names to something like:
abcd
abcde

To avoid suffix and prefix match with 'bc' search

Reply:
Sure! 'bc' makes more sense. It should generate same output.